app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir=. --presets=es2015",
```

Comment:
remove `--presets=es2015` from this line.

Add the following to . `package.json`:

```json
{
  "babel": {
    "presets": ["es2015"]
  },
  "ava": {
    "require": ["babel-register"]
  },
  "xo": {
    "esnext": true
  }
}
```

I also thing `--out-dir` should be `dist`. XO ignores that by default, and it's generally understood that contains compiled code.